1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -66,11 +64,11 @@ You can override the default network or add additional overlay networks by modif
           prefix: 24
 ```
```

Comment:
yes, if he installs manually. This is configurable in the config.yaml file. It is not accessible if you deploy with a standard cloud template (in which case the default "dcos" network is included", but it is accessible if you just deploy manually configuring the file (locally on in the cloud)